web/src/queries/zfcp.ts

```diff
@@ -26,67 +26,56 @@ import { useInstallerClient } from "~/context/installer";
 import React from "react";
 import { ZFCPConfig, ZFCPController, ZFCPDisk } from "~/types/zfcp";
 
-/**
- * Returns a query for retrieving the zFCP controllers
- */
-const zfcpControllersQuery = () => ({
+const zfcpControllersQuery = {
```

Comment:
BTW, maybe I miss something at some point but the only suggestion I see is about the naming: https://github.com/openSUSE/agama/pull/1570#pullrequestreview-2302148845

But not about the full structure.  I've no strong opinion about writing them as function or object. But I though we were writing them always as function for consistence since they could receive arg for building the query. As far as I can see, @imobachgs has wrote some as function and some as object in the storage part too. We can go ahead with such approach if you think it is better, but we should be consistent in all places.

PLEASE, do not take this comment as a suggestion for continue sending changes against this PR. Whatever we decide to choose should be addressed in a change unifying all the queries across the code. Otherwise we will be just adding more and more noise here.

Reply:
This change was based on this another suggestion https://github.com/openSUSE/agama/pull/1570#discussion_r1758205470, in that case objects were used, so, I also does not have an strong opinion and agree on unifying.

Reply:
> This change was based on this another suggestion [#1570 (comment)](https://github.com/openSUSE/agama/pull/1570#discussion_r1758205470), in that case objects were used, so, I also does not have an strong opinion and agree on unifying.

Well, the point of my comment was not the `queryKey` thingie, but the overall approach of the whole function. I insisted in https://github.com/openSUSE/agama/pull/1570#discussion_r1764431153.